src/openforms/dmn/contrib/camunda/plugin.py

```diff
@@ -0,0 +1,121 @@
+        return ""
+
+    return results[0]["id"]
+
+
+def handle_camunda_error(error: requests.HTTPError):
+    logger.exception("Error occurred while calling Camunda API", exc_info=error)
+    if error.response is None:
+        raise
+
+    try:
+        response_body = error.response.json()
+    except (json.JSONDecodeError, simplejson.JSONDecodeError):
+        logger.exception("Could not decode JSON data in error response body")
+    else:
+        logger.error("Camunda error information: %r", response_body)
+
+
+@register("camunda")
+class Plugin(BasePlugin):
+    verbose_name = _("Camunda")
+
+    @staticmethod
+    def get_available_decision_definitions() -> List[DecisionDefinition]:
+        with get_client() as client:
+            results = client.get(
+                "decision-definition",
+                params={"latestVersion": "true"},
+            )
+        return [
+            DecisionDefinition(
+                identifier=result["key"], label=result["name"] or result["key"]
+            )
+            for result in results
+        ]
+
+    @staticmethod
+    def evaluate(
+        definition_id: str, *, version: str = "", input_values: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        with get_client() as client:
+            camunda_id = _get_decision_definition_id(client, definition_id, version)
+            try:
+                result = evaluate_dmn(
+                    dmn_key=definition_id,
+                    dmn_id=camunda_id,
+                    input_values=input_values,
+                    client=client,
+                )
+            except requests.HTTPError as error:
+                handle_camunda_error(error)
+                return {}
+        return result
+
+    @staticmethod
+    def get_decision_definition_versions(
+        definition_id: str,
+    ) -> List[DecisionDefinitionVersion]:
+        """
+        Get a collection of available versions for a given decision definition.
+
+        :param definition_id: the key of the decision definition in the Camunda API.
+        """
+        with get_client() as client:
+            results = client.get(
+                "decision-definition",
+                params={"key": definition_id, "sortBy": "version", "sortOrder": "desc"},
+            )
+        return [
+            DecisionDefinitionVersion(
+                id=str(result["version"]),
+                label=_("v{version} (version tag: {version_tag})").format(
+                    version=result["version"],
+                    version_tag=result["version_tag"] or _("n/a"),
+                ),
+            )
+            for result in results
+        ]
+
+    @staticmethod
+    def get_definition_xml(definition_id: str, version: str = "") -> str:
+        with get_client() as client:
+            camunda_id = _get_decision_definition_id(client, definition_id, version)
+            xml_response = client.get(f"decision-definition/{camunda_id}/xml")
+        return xml_response["dmn_xml"]
```

Comment:
Should have asked in previous PR but is `check_config` implemented? I thought I saw it but would have expected it here.

Reply:
ah no, I forgot about it when porting the hackathon code!

Reply:
Added that in #1683